dataedit/static/dataedit/backend.js

```diff
@@ -262,21 +262,44 @@ function load_view(schema, table, csrftoken, current_view) {
     $.when(
         $.ajax({
             url: '/api/v0/schema/' + schema + '/tables/' + table + '/columns',
-        }), $.ajax({
+        }),
+        $.ajax({
             type: 'POST',
             url: '/api/v0/advanced/search',
             dataType: 'json',
             data: {
                 csrfmiddlewaretoken: csrftoken,
                 query: JSON.stringify(count_query)
             }
+        }),
+        $.ajax({ // get metadata for column description / units
+            url: '/api/v0/schema/' + schema + '/tables/' + table + '/meta'
         })
-    ).done(function (column_response, count_response) {
-
-
+    ).done(function (column_response, count_response, meta) {
+        var columnDescription = {}; // name -> description
+        try {
+            meta[0]['resources'][0]['schema']['fields'].map(function(fld){
+                columnDescription[fld.name] = fld.description || "";
```

Comment:
As far as I understand this would fail if a single field does not have a "name" property. This may happen due to a faulty metadata string and should be caught.

Reply:
no, that would not throw an error, it would only set columnDescription[undefined]

Reply:
i guess it would only break if fld is not an object